code/modules/item_worth/materials.dm

```diff
@@ -2,64 +2,64 @@
 	var/value = 1
 
 /material/uranium
-	value = 100
+	value = 75
```

Comment:
Did you check that this actually works like we think it does? someone raised doubts